modules/logger.lua

```diff
@@ -138,19 +139,18 @@ end
 ---@field source string source of the log. Usually a playerId or name of a resource.
 ---@field event string the action or 'event' being logged. Usually a verb describing what the name is doing. Example: SpawnVehicle
 ---@field message string the message attached to the log
----@field webhook? string Discord logs only. url of the webhook this log should send to
+---@field webhook? string Discord logs only. url of the webhook this log should send to. looks up via config
 ---@field color? string Discord logs only. what color the message should be
----@field tags? string[] Discord logs only. tags in discord. Example: ['@admin', '@everyone']
+---@field tags? string[] Discord logs only. tags in discord. Example: {'<@%roleid>', '@everyone'}
 
 ---Creates a log using either ox_lib logger, discord webhooks depending on convar
```

Comment:
```suggestion
---Logs using ox_lib if ox_lib logging is configured. Additionally logs to discord if a web hook is passed.
```
Wanting to clarify this AND behavior of logging to both.